DTS-STN/dts-rmp
# 235 date format

# Description

Formatted date inside Engagement detail view yyyy-mm-dd. 
Formatted the engagement detail view page to match the same margins as contact detail view page.
Fixed the contact detail view as it was changed to NOT use contact.name in lang files. (Same for address, email and phone.

## Test Instructions

1. Go to any engagement detail page. Notice the date and the margins on full width (1200px or more). It is the same as any contact detail view page.
2. go to any contact detail view page. Notice how each label is now 'Contact (example: Name)' or 'Organisation (example: Address)'  

## Help Requested

For the contact detail view page. I followed the mock-up when I created it and it only has 2 organisation information fields. Should we add more? or leave it like that?

## Checklist
